roles/commcare_analytics/defaults/main.yml

```diff
@@ -89,7 +89,7 @@ certbot_domains:
 # Database backup variables
 postgresql_backup_enabled: "{{ enable_postgresql_backup }}"
 postgresql_backup_dir: "/var/tmp"
-postgresql_backup_db_list: "superset superset_meta superset_oauth2"
+postgresql_backup_db_list: "superset_meta superset_oauth2"
```

Comment:
@Charl1996 Can you please confirm this list of dbs looks right to you? I believe these 2 are the ones you intended to be backed up?

Reply:
Yes, I think that looks right. The `superset_cchq_data` database (fetched from `get_hq_database` in code) stores the imported data, so I don't think we need to backup those since HQ is the source of truth for that.

I'm can't remember what the contents of `superset` database contain, but suspect that's superset-related things that we don't need to backup. But it does not exist on staging which to me is slightly weird.

Reply:
Oh, I see your full commit message now. Seems ok to omit `superset` database from the list.